types/react-big-calendar/react-big-calendar-tests.tsx

```diff
@@ -16,16 +16,16 @@ class CalendarEvent {
     title: string;
     allDay: boolean;
     start: Date;
-    end: Date;
```

Comment:
Why rename this? Did `end` clash with something as an identifier?

Reply:
I arbitrarily chose one of the fields of the custom `CalendarEvent` to be different from the default `Event` interface definition.
Before this PR, it would turn into a Typescript error, complaining that `CalendarEvent` doesn't match `Event`. However, this is a valid example and shouldn't fail.
I modified the example to prevent a regression error on future updates to the declaration file.